handlers.go

```diff
@@ -55,9 +56,9 @@ func (r *oauthProxy) getRedirectionURL(w http.ResponseWriter, req *http.Request)
 		redirect = r.config.RedirectionURL
 	}
 
-	state, _ := req.Cookie("OAuth_Token_Request_State")
+	state, _ := req.Cookie(requestStateCookie)
 	if state != nil && req.URL.Query().Get("state") != state.Value {
-		r.log.Error("State parameter mismatch")
+		r.log.Error("state parameter mismatch")
```

Comment:
Out of curiosity, why you removed the capital letter at the start of each error message? Any specific reason? If there's no reason, my 2 cents is to leave it as is.

Reply:
this is one of these linting rules